doc/doc-support/python-interpreter-table/collect-data/interpreters.nix

```diff
@@ -0,0 +1,44 @@
+/* Collect Python interpreters from attrset `pkgs.pythonInterpreters`.
+
+The return type is an attrset with the following shape:
+```
+{
+  ${interpreterFieldValue} = {
+    interpreter = ${interpreterFieldValue};
+    pkgKey = ${pkgKey};
```

Comment:
I have never seen to the attr of a package being referred to as pkgKey. We should probably use attrPath or so instead.